victoriametrics/Dockerfile

```diff
@@ -33,7 +33,7 @@ COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/bin/
 COPY --from=build-single /go/src/github.com/VictoriaMetrics/VictoriaMetrics/LICENSE /
 
 USER 10000:10000
-EXPOSE 8428
```

Comment:
Why did you change the exposed port?

Reply:
It is from upstream changing.

Reply:
victoria-metrics
curl -f -s -L -o CURRNT_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.98.0-cluster/app/victoria-metrics/deployment/Dockerfile
curl -f -s -L -o UPDATE_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.100.1-cluster/app/victoria-metrics/deployment/Dockerfile
--- CURRNT_VER  2024-04-15 05:45:36.775548489 +0000
+++ UPDATE_VER  2024-04-15 05:45:37.067548365 +0000
@@ -1,7 +1,7 @@
 ARG base_image
 FROM $base_image
 
-EXPOSE 8428
+EXPOSE 9428

Reply:
I could not open that URLs.

I think it is VictoriaLogs Dockerfile?
https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6082

Reply:
@umezawatakeshi @zoetrope 
Sorry, it is my mistake.
the pot changing is only victoria-logs.

curl -f -s -L -o CURRNT_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.98.0-cluster/app/victoria-logs/deployment/Dockerfile
curl -f -s -L -o UPDATE_VER https://raw.githubusercontent.com/VictoriaMetrics/VictoriaMetrics/v1.100.1-cluster/app/victoria-logs/deployment/Dockerfile
--- CURRNT_VER  2024-04-15 05:45:36.775548489 +0000
+++ UPDATE_VER  2024-04-15 05:45:37.067548365 +0000
@@ -1,7 +1,7 @@
 ARG base_image
 FROM $base_image
 
-EXPOSE 8428
+EXPOSE 9428

I will update currect port no